.travis.yml

```diff
@@ -33,5 +34,5 @@ notifications:
   slack:
     secure: eX3VgtKQ48rzKGdXpIZZVYINffI2wbqhqoJFUxHO1Zku6tXzbt0R+r3NVHo37wBID63bkXjYqjZuv+JRUtvf/XO51+QkTqCCZ6iC3lr6IEqRpzxXsmNYf+QPhuN2kvVTSyycHFXdwQLL7sCniFSmEcLdu6xMpobI4PUwi2OvHj4=
 before_install:
-- openssl aes-256-cbc -K $encrypted_6a9d32f3e0bd_key -iv $encrypted_6a9d32f3e0bd_iv
-  -in ci.enc -out testing/ci -d
+  - if [ -z "$ARE_SECRETS_AVAILABLE" ]; then MANTL_CI_FORK_CHECK=1; else MANTL_CI_FORK_CHECK=0; fi
```

Comment:
Yes. I added it in the Settings.
> On May 12, 2016, at 14:50, Brian Hicks <notifications@github.com> wrote:
> 
> In .travis.yml <https://github.com/CiscoCloud/mantl/pull/1415#discussion_r63085135>:
> 
> > @@ -33,5 +34,5 @@ notifications:
> >    slack:
> >      secure: eX3VgtKQ48rzKGdXpIZZVYINffI2wbqhqoJFUxHO1Zku6tXzbt0R+r3NVHo37wBID63bkXjYqjZuv+JRUtvf/XO51+QkTqCCZ6iC3lr6IEqRpzxXsmNYf+QPhuN2kvVTSyycHFXdwQLL7sCniFSmEcLdu6xMpobI4PUwi2OvHj4=
> >  before_install:
> > -- openssl aes-256-cbc -K $encrypted_6a9d32f3e0bd_key -iv $encrypted_6a9d32f3e0bd_iv
> > -  -in ci.enc -out testing/ci -d
> > +  - if [ -z "$ARE_SECRETS_AVAILABLE" ]; then MANTL_CI_FORK_CHECK=1; else MANTL_CI_FORK_CHECK=0; fi
> where is this environment variable coming from? Travis?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/CiscoCloud/mantl/pull/1415/files/89c1b3d3c65fa0ac9d1a205342ee47aa9e629995#r63085135>